integration/src/main/java/dev/ikm/tinkar/integration/DataIntegrity.java

```diff
@@ -75,7 +75,7 @@ public static void startup(File datastoreRootLocation){
         LOG.info("Datastore location: " + datastoreRootLocation);
         CachingService.clearAll();
         ServiceProperties.set(ServiceKeys.DATA_STORE_ROOT, datastoreRootLocation);
-        PrimitiveData.selectControllerByName(TestConstants.SA_STORE_OPEN_NAME);
+        PrimitiveData.selectControllerByName("Open SpinedArrayStore");
```

Comment:
Why are we going away from using a constant here?

Reply:
@dettmoney, the constant was removed in preference of an enum within the test structure. This DataIntegrity class should not live here and will be moved elsewhere in another 'part' of this update to the data integration tests.